TinyXml/src/tinyxmlparser.cpp

```diff
@@ -334,12 +334,12 @@ const TCHAR* TiXmlBase::ReadText(	const TCHAR* p,
 		while (	   p && *p
 				&& !StringEqual( p, endTag, caseInsensitive ) )
 		{
-			if ( *p == '\r' || *p == '\n' )
+			if ( *p == L'\r' || *p == L'\n' )
```

Comment:
Use `TEXT('\r')` and `TEXT('\n')` as `*p` is of type `TCHAR`.

Reply:
@MAPJe71 As PluginManager just supports the unicode build that makes no difference as TEXT() equals in this case L, but for a clean codebase and to be in sync with the surrounding code I have corrected that with https://github.com/bruderstein/nppPluginManager/commit/1a02cf1e8a1fcc8f7f4d2528f45d0270c3b5a6a6

Thanks for the review.